products/product_analytics/manifest.tsx

```diff
@@ -137,7 +139,7 @@ export const manifest: ProductManifest = {
     ],
     treeItemsProducts: [
         {
-            path: 'Product analytics',
+            path: 'Product analytics', // Keep in sync with PRODUCT_ANALYTICS_PRODUCT_TREE_NAME
```

Comment:
Maybe I can drop it and just use the string constants on `frontend/src/layout/panel-layout/ProjectTree/projectTreeWebAnalyticsExperiment.tsx`.

The imports were failing and I did find a way to make the `product.ts` generation work with them, but it seemed to be too much hassle for a simple change.